From 5afc438bb66fe32450a92135fdcce4bff60afd7d Mon Sep 17 00:00:00 2001
From: Bryan O'Donoghue <bryan.odonoghue@linaro.org>
Date: Tue, 30 Jul 2019 12:59:53 +0100
Subject: [PATCH 1/3] core: imx: imx6ul: Define the base address of UART6

This patch defines the base address of UART6 on the i.MX6 series.

Signed-off-by: Bryan O'Donoghue <bryan.odonoghue@linaro.org>
---
 core/arch/arm/plat-imx/imx-regs.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-imx/imx-regs.h b/core/arch/arm/plat-imx/imx-regs.h
index e1dafff..c577865 100644
--- a/core/arch/arm/plat-imx/imx-regs.h
+++ b/core/arch/arm/plat-imx/imx-regs.h
@@ -56,6 +56,7 @@
 #define UART3_BASE			0x021EC000
 #define UART4_BASE			0x021F0000
 #define UART5_BASE			0x021F4000
+#define UART6_BASE			0x021FC000
 #define AIPS1_BASE			0x02000000
 #define AIPS1_SIZE			0x100000
 #define AIPS2_BASE			0x02100000
-- 
2.7.4

